clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -708,6 +745,28 @@ protected Node curNode() {
             return curNode;
         }
 
+        public RetryContext getRetryContext() {
```

Comment:
nit: Same comment as previously, normally the convention is to drop the `get` in front of getter method names.

Reply:
Refactored